MetaMask/metamask-extension
# Update `depcheck` to latest version

`depcheck` has been updated to the latest version. This version pins `@babel/parser` to v7.16.4 because of unresolved bugs in v7.16.5 that result in `depcheck` failing to parse TypeScript files correctly ([see here for details](https://github.com/depcheck/depcheck/issues/688#issuecomment-1002121684))

We had a Yarn resolution in place to ensure `@babel/parser@7.16.4` was being used already. That resolution is no longer needed so it has been removed. This should resove the issue the dev team has been seeing lately where `yarn` and `yarn-deduplicate` disagree about the state the lockfile should be in.

## Manual Testing Steps

Run `yarn depcheck`, and see that it completes successfully without flagging any issues.

## Pre-Merge Checklist


### + If there are functional changes:
